Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Controllers/NoticestoMarinersControllerTest.cs

```diff
@@ -32,38 +34,79 @@ public void Setup()
         }
 
         [Test]
-        public void WhenIndexIsCalled_ThenShouldReturnsExpectedView()
+        public async Task WhenIndexIsCalled_ThenShouldReturnsExpectedView()
         {
-            const string expectedView = "~/Views/NoticesToMariners/FilterWeeklyFiles.cshtml";
-            IActionResult result = _controller.Index();
+            const string expectedView = "~/Views/NoticesToMariners/Index.cshtml";
+
+            A.CallTo(() => _fakeNMDataService.GetWeeklyFilesResponseModelsAsync(A<int>.Ignored, A<int>.Ignored, A<string>.Ignored));
+
+            IActionResult result = await _controller.Index();
             Assert.IsInstanceOf<ViewResult>(result);
             string actualView = ((ViewResult)result).ViewName;
             Assert.AreEqual(expectedView, actualView);
         }
 
         [Test]
-        public void WhenDailyFilesIsCalled_ThenShouldReturnsExpectedView()
+        public async Task WhenIndexPostIsCalled_ThenShouldReturnsExpectedViewAndViewData()
         {
-            const string expectedView = "~/Views/NoticesToMariners/ShowDailyFiles.cshtml";
-            IActionResult result = _controller.DailyFiles();
+            const string expectedView = "~/Views/NoticesToMariners/Index.cshtml";
+            const int year = 2022, week = 20;
```

Comment:
Please split these declarations onto separate lines:
```suggestion
            const int year = 2022;
            const int week = 20;
```

Reply:
Implemented declarations onto separate lines